san-francisco-tourism/build.gradle

```diff
@@ -0,0 +1,33 @@
+
+group = 'com.oracle.javaone.2017.hol.junit5'
+version = '0.0.1-SNAPSHOT'
+
+description = "junit5 with gradle as build system"
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+ext.junitVintageVersion  = '5.0.0'
+ext.junitPlatformVersion = '1.2.0'
+ext.junitVersion  = '5.0.0'
+
+
+repositories {
+        
+     maven { url "http://repo.maven.apache.org/maven2" }
+}
+dependencies {
+    compile group: 'org.mockito', name: 'mockito-core', version:'2.9.0'
+    testCompile group: 'junit', name: 'junit', version:'4.12'
+    testCompile group: 'org.hamcrest', name: 'hamcrest-all', version:'1.3'
+
+    testCompile group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: "$junitVersion"
```

Comment:
are these dependencies for the solution, rather than the exercise (e.g. where we ant to get to)?